<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataUI</class>
 <widget class="QWidget" name="dataUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dataUI</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>891</width>
     <height>781</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="QueryBox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>340</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Query</string>
   </property>
   <widget class="QPushButton" name="IncButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QueryInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_pb">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WeekRP_pb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Weekly Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Monthrp_pb">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Monthly Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadNew">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateRec">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Record</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SimulationBox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Simulation Control</string>
   </property>
   <widget class="QPushButton" name="SimruntButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="RunLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stopped</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Conditionset_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QListWidget" name="NoteList">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>110</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="NowDisp">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1970/01/01/00/00/00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Step_pb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>SimruntButton</tabstop>
  <tabstop>Step_pb</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Conditionset_button</tabstop>
  <tabstop>NoteList</tabstop>
  <tabstop>QueryInput</tabstop>
  <tabstop>LoadNew</tabstop>
  <tabstop>IncButton</tabstop>
  <tabstop>Clear_pb</tabstop>
  <tabstop>UpdateRec</tabstop>
  <tabstop>WeekRP_pb</tabstop>
  <tabstop>Monthrp_pb</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
